docs/06_how-to-guides/40_multi-index/how-to-define-a-primary-index.md

```diff
@@ -10,13 +10,13 @@ This guide provides instructions to define a primary index for a multi-index tab
 
 See the following code reference:
 
-* The [`multi-index`](../../classeosio_1_1multi__index) class.
+* The [`multi-index`](../../reference/Classes/classeosio_1_1multi__index) class.
```

Comment:
I was on your branch, but got 404 when I click
https://github.com/AntelopeIO/cdt/blob/fix-devrel%23204-3.0/docs/reference/Classes/classeosio_1_1multi__index

Reply:
Those links are relative to the deployed docs portal site: https://docs.eosnetwork.com/cdt/latest/
In the above case it will resolve correctly to: https://docs.eosnetwork.com/cdt/latest/reference/Classes/classeosio_1_1multi__index/

Reply:
Oh, I see.